docs/src/app.css

```
@@ -188,6 +268,11 @@
 
     body {
         @apply bg-background text-foreground;
+        font-family:
+            'lato',
+            system-ui,
+            -apple-system,
+            sans-serif;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)